### PR TITLE
refactor: implement Angular control flow for `forms` library

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**
[AB#2992826](https://dev.azure.com/blackbaud/f565481a-7bc9-4083-95d5-4f953da6d499/_workitems/edit/2992826)

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/2648/storybook/)

Component Storybooks:

- [action-bars-storybook](https://blackbaud.github.io/skyux-pr-preview/2648/storybooks/action-bars-storybook/)
- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/2648/storybooks/ag-grid-storybook/)
- [angular-tree-component-storybook](https://blackbaud.github.io/skyux-pr-preview/2648/storybooks/angular-tree-component-storybook/)
- [avatar-storybook](https://blackbaud.github.io/skyux-pr-preview/2648/storybooks/avatar-storybook/)
- [colorpicker-storybook](https://blackbaud.github.io/skyux-pr-preview/2648/storybooks/colorpicker-storybook/)
- [data-manager-storybook](https://blackbaud.github.io/skyux-pr-preview/2648/storybooks/data-manager-storybook/)
- [datetime-storybook](https://blackbaud.github.io/skyux-pr-preview/2648/storybooks/datetime-storybook/)
- [errors-storybook](https://blackbaud.github.io/skyux-pr-preview/2648/storybooks/errors-storybook/)
- [forms-storybook](https://blackbaud.github.io/skyux-pr-preview/2648/storybooks/forms-storybook/)
- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/2648/storybooks/indicators-storybook/)
- [inline-form-storybook](https://blackbaud.github.io/skyux-pr-preview/2648/storybooks/inline-form-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/2648/storybooks/layout-storybook/)
- [lists-storybook](https://blackbaud.github.io/skyux-pr-preview/2648/storybooks/lists-storybook/)
- [lookup-storybook](https://blackbaud.github.io/skyux-pr-preview/2648/storybooks/lookup-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/2648/storybooks/pages-storybook/)
- [phone-field-storybook](https://blackbaud.github.io/skyux-pr-preview/2648/storybooks/phone-field-storybook/)
- [split-view-storybook](https://blackbaud.github.io/skyux-pr-preview/2648/storybooks/split-view-storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/2648/storybooks/tabs-storybook/)
- [text-editor-storybook](https://blackbaud.github.io/skyux-pr-preview/2648/storybooks/text-editor-storybook/)
- [theme-storybook](https://blackbaud.github.io/skyux-pr-preview/2648/storybooks/theme-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/2648/code-examples/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/2648/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/2648/playground/)

File: libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.html [hunk 1]
```diff
@@ -35,29 +35,31 @@
...
-    </label>
+    @if (hasLabelComponent || labelText) {
+      <label
+        class="sky-toggle-switch-label"
+        [for]="toggle.id"
+        [attr.id]="labelId"
+      >
+        @if (labelText) {
+          @if (!labelHidden) {
+            <ng-container>{{ labelText }}</ng-container>
```
Comment (Blackbaud-SteveBrush) [inline libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.html:46]: I think you can remove the `ng-container` now.

File: libs/components/forms/src/lib/modules/character-counter/character-counter-indicator.component.html [hunk 2]
```diff
@@ -5,21 +5,19 @@
...
-  <ng-template #screenReaderCountMessage>
-    <ng-container
-      *ngIf="
-        characterCount
-          | skyCharacterCounterScreenReader
-            : characterCountLimit as screenReaderCount
-      "
-      >{{
+  @if (characterCount > characterCountLimit) {
+    <ng-container>{{
```
Comment (Blackbaud-SteveBrush) [inline libs/components/forms/src/lib/modules/character-counter/character-counter-indicator.component.html:9]: `ng-container` can be removed.

File: libs/components/forms/src/lib/modules/character-counter/character-counter-indicator.component.html [hunk 3]
```diff
@@ -5,21 +5,19 @@
...
+      'skyux_character_count_over_limit' | skyLibResources
+    }}</ng-container>
+  } @else {
+    @if (
+      characterCount | skyCharacterCounterScreenReader: characterCountLimit;
+      as screenReaderCount
+    ) {
+      <ng-container>{{
```
Comment (Blackbaud-SteveBrush) [inline libs/components/forms/src/lib/modules/character-counter/character-counter-indicator.component.html:17]: Ditto.

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**